geocity/apps/submissions/views.py

```diff
@@ -1615,7 +1615,7 @@ def submission_media_download(request, property_value_id):
 @method_decorator(login_required, name="dispatch")
 @method_decorator(check_mandatory_2FA, name="dispatch")
 @method_decorator(permanent_user_required, name="dispatch")
-class SubmissionList(ExportMixin, SingleTableMixin, FilterView):
+class SubmissionList(PandasExportMixin, SingleTableMixin, FilterView):
```

Comment:
I guess we should leave a option to the user to use the "old" excel export, thus adapt the mixing dynamically depending on a specific action ?

Reply:
Old excel export is still usefull ?